### DISCUSSION TITLE
Feature Request: Extra Field "Prerequisites" for each Talk

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1750] [Q&A] [answered]
Hi,

For our events it would be nice to offer a Field "Prerequisites" for each Talk, so that the Speakers can write additional infos for the level of the talk.
(We have Workshops not Talks and it would be nice to have written Prerequisites, so that the participiants can better estimate if the workshop is suiteable for them or not)

Thanks a lot!

**Comment by rixx** [comment, marked as answer]
pretalx offers organisers the option to add as many custom fields ("questions") as they want, which may be of the type "text" (for free-form input) or "choose [multiple] from a list" (if you want structured feedback). Including "prerequisites" as a standard field would not make sense for many conferences though, so configuring this as a question would be our recommended way to go.

**Reply by navlisilvan** [reply]
Ah, cool, thanks.

The answer to such a question does not appear in the JSON or XML export of the plan, does it?

**Reply by navlisilvan** [reply]
`"answers": []`
I only found this empty tag in a test talk (and every Talk)

**Reply by rixx** [reply]
If answers are included in the API depends on your permissions and the state of the question (i.e. if both the question and the talk are public, if you're using an auth token, etc).

**Reply by navlisilvan** [reply]
Ah, when i export it from the backend, the Question and Answer is included. When I export it from the frontend, it is not included.
Is there an option to enable this for the frontend export? 
In the "frontend web view", when you click on the Talk, the answer ist already visible.

**Reply by rixx** [reply]
If you go to "Proposals -> Export", all questions should appear as checkboxes to select to be included in your JSON or CSV export.
